apps/aevm/test/VMTests/test_generator.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Do we need the generator?

Reply:
Probably not, but we might want to repeat the stunt (for some other tests) at some point?